### ISSUE TITLE
Search has become very slow with huge collections now in the index

### ISSUE THREAD

**Issue by burnash** [neurolearn/neurolearn-web#17]
The Input Data page now loads about 7MB of JSON before rendering. This slows down the page load dramatically. One way of fixing this is to split ES index and move images in their own ElasticSearch index or do not store images data in ES at all.

**Comment by ljchang** [comment]
I've noticed this too.  I'm not too sure how the elastic search works.  Do the images need to be stored in the ES index?  We aren't ever searching for the image itself, right?  Could we just have pointers to the files stored on the file system or to neurovault thumbnail that gets rendered when the page loads?

**Comment by burnash** [comment]
I reworked the search and removed image data from the collection search results. Now the search runs much faster. The app loads image data from NeuroVault's API before displaying the select images dialog. So when the user continues to the training label selection page the data is already loaded.

There's a problem with inconsistent results from the NeuroVaults API — some images are missing in the API response, I already submitted a fix for this: https://github.com/NeuroVault/NeuroVault/pull/566
